docs/diagnostics/PX1040.md

```diff
@@ -8,7 +8,7 @@ This document describes the PX1040 diagnostic.
 | PX1040 | Constructors in BLC extensions are strictly prohibited. You should use the `Initialize()` method instead. | Error | Available |
 
 ## Diagnostic Description
-Constructors cannot be used in graph extensions. Instead of using constructors, you must override the `Initialize()` method.
+Constructors cannot be used in graph extensions. Instead of using instance constructors, you must override the `Initialize()` method.
```

Comment:
The first sentence means that any constructors cannot be used in graph extensions, but static constructors are allowed.
Correction in the second sentence is correct.